src/main/scala/satify/model/dpll/PartialModel.scala

```diff
@@ -0,0 +1,12 @@
+package satify.model.dpll
+
+import satify.model.Variable
+
+object OrderedSeq:
+  def seq[T](elems: T*)(using ordering: Ordering[T]): Seq[T] = elems.distinct.sorted(ordering)
+
+given Ordering[Variable] with
+  def compare(x: Variable, y: Variable): Int = (x, y) match
+    case (Variable(xName, _), Variable(yName, _)) => xName.compareTo(yName)
```

Comment:
I would put the given inside the object and then from outside I would use `import satify.model.dpll.OrderedSeq.given`
What do you think?